plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitPresenter.java

```diff
@@ -259,30 +267,11 @@ public void onSelectionChanged(Path path, boolean isChecked) {
 
   @Override
   public void setAmendCommitMessage() {
-    service
-        .log(project.getLocation(), null, -1, -1, false)
-        .then(
-            log -> {
-              String message = "";
-              final Revision revision = getFirst(log.getCommits(), null);
-              if (revision != null) {
-                message = revision.getMessage();
-              }
-              CommitPresenter.this.view.setMessage(message);
-              CommitPresenter.this.view.setEnableCommitButton(!message.isEmpty());
-            })
-        .catchError(
-            error -> {
-              if (getErrorCode(error.getCause()) == ErrorCodes.INIT_COMMIT_WAS_NOT_PERFORMED) {
-                dialogFactory
-                    .createMessageDialog(
-                        constant.commitTitle(), constant.initCommitWasNotPerformed(), null)
-                    .show();
-              } else {
-                CommitPresenter.this.view.setMessage("");
-                notificationManager.notify(constant.logFailed(), FAIL, NOT_EMERGE_MODE);
-              }
-            });
+    if (latestCommitMessage == null) {
```

Comment:
Just set focus to commit message text box

Reply:
Added setting focus to message field on clicking the `amend` checkbox.